pxtlib/cpp.ts

```diff
@@ -1367,8 +1381,10 @@ namespace pxt.hexloader {
                                             }
                                         },
                                             e => {
-                                                setTimeout(tryGet, 1000)
-                                                return null
+                                                delay = Math.min(30000, delay * 1.5); // exponential back off, max 30sec
```

Comment:
```suggestion
                                                delay = Math.min(30000, delay * 2); // exponential back off, max 30sec
```

possibly, or maybe just a few more retries? at 1.5* and 7 max retries it would time out after ~48seconds with a max delay of 17s, I think 48s is still in the realm of "possibly going to succeed"

Reply:
Removed backoff